src/Codeception/Util/XmlBuilder.php

```diff
@@ -69,49 +68,48 @@
  *  * `$xml->getDom` - get a DOMDocument object
  *  * `$xml->__toString` - get a string representation of XML.
  *
- * [Source code](https://github.com/Codeception/Codeception/blob/4.0/src/Codeception/Util/XmlBuilder.php)
```

Comment:
I promised to @ThomasLandauer to automatically generate Edit links to the source of autogenerated reference pages.
I can't find where I did that :D

Reply:
You mean where you promised it? Here: https://github.com/Codeception/Codeception/discussions/6063#discussioncomment-245077